resources/scripts/components/server/files/FileManagerBreadcrumbs.tsx

```diff
@@ -36,7 +36,7 @@ export default ({ renderLeft, withinFileEditor, isNewFile }: Props) => {
         });
 
     return (
-        <div css={tw`flex flex-grow-0 items-center text-sm text-neutral-500 overflow-x-hidden`}>
+        <div css={tw`flex flex-grow-0 items-center text-sm text-neutral-500`}>
```

Comment:
Why was this change made?

Reply:
The directory path would be masked by the input field if ``overflow-x-hidden`` was present.